flask_stupe/__init__.py

```diff
@@ -20,6 +20,11 @@
 except ImportError:  # pragma: no cover
     pymongo = False
 
+try:
+    from pymongo.collation import Collation
+except ImportError:  # pragma: no cover
+    Collation = False
+
```

Comment:
I'd suggest to rather combine it with the previous block (see how it's done for `dateutil`, for example).

Reply:
Done